app/uk/gov/hmrc/customs/declaration/services/CommunicationService.scala

```diff
@@ -92,7 +89,6 @@ class CommunicationService @Inject()(logger: DeclarationsLogger,
     maybeXClientId.fold[Future[Option[String]]](Future.successful(None)) { xClientId =>
       val apiSubscriptionKey = ApiSubscriptionKey(xClientId, apiContextEncoded, requestedApiVersionNumber)
       apiSubFieldsConnector.getSubscriptionFields(apiSubscriptionKey) map (response => {
-        logger.debug("got api-subscription-fields response", payload = response.toString)
         Some(response.fieldsId.toString)
```

Comment:
debug should be added in the connector or the service calling the connector, in case of failure that should log the error, also it should log all the data we are sending to MDG at debug level which is one of the requirements in the story.